README.md

```diff
@@ -15,53 +15,85 @@
    </a>
 </p>
 
-## Requirements
+## Build requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8, 64-bit
 * Python 2.6+ (for running with the launcher script)
-* Docker (for building and running tests)
+* Docker (for building and running tests, and for building documentation)
 
 ## Building Presto
 
-Presto is a standard Maven project. Simply run the following command from the project root directory:
+Presto is a standard Maven project. Simply run the following command from the
+project root directory:
 
     ./mvnw clean install
 
-On the first build, Maven will download all the dependencies from the internet and cache them in the local repository (`~/.m2/repository`), which can take a considerable amount of time. Subsequent builds will be faster.
+On the first build, Maven downloads all the dependencies from the internet
+and caches them in the local repository (`~/.m2/repository`), which can take a
+considerable amount of time. Subsequent builds are faster.
 
-Presto has a comprehensive set of unit tests that can take several minutes to run. You can disable the tests when building:
+Presto has a comprehensive set of unit tests that can also take a considerable
+amount of time to run. You can disable the tests when building:
 
     ./mvnw clean install -DskipTests
 
+If you experience a build fail, in many cases you can resume the build once you
```

Comment:
Let's remove this part, since removing `clean` can be dangerous and doesn't really help anyway